pkg/manifest/os.go

```diff
@@ -406,6 +409,10 @@ func (p *OS) getBuildPackages(distro Distro) []string {
 		// version of uki-direct. Add it conditioned just on the bootloader
 		// type for now, until we find a better way to decide.
 		packages = append(packages, "libkcapi-hmaccalc")
+
+		// UKI-based images set a dnf versionlock on shim, so we need to
+		// include the plugin.
+		packages = append(packages, "python3-dnf-plugin-versionlock")
```

Comment:
Note that this is needed for DNF4 version lock (so RHEL and CentOS Stream), but not required on Fedora with DNF5, which has the command built in.